backend/graminsta/post/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+Urls for post app.
+"""
+
+from django.urls import path
+from .views import PostRecordView
+
+
+app_name = 'post'
+urlpatterns = [
+    path('createPost/', PostRecordView.as_view(), name='post'),
```

Comment:
Don't use `createPost`, instead, use `create-post`. As per Google search guidelines:

> Consider using punctuation in your URLs. The URL http://www.example.com/green-dress.html is much more useful to us than http://www.example.com/greendress.html. We recommend that you use hyphens (-) instead of underscores (_) in your URLs.

See also: https://support.google.com/webmasters/answer/76329?hl=en